frontend/package.json

```diff
@@ -10,9 +10,9 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "antd": "^5.11.5",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "antd": "^5.11.5",
```

Comment:
these got moved out of alphabetical order?

Reply:
:o